notebooks/working/_03-tb-deconfounder-predictive-checks.py

```diff
@@ -505,10 +276,6 @@ def perform_visual_predictive_cit_test(
     norm(loc=z_post_params['z_mean_inferred'].values,
          scale=z_post_params['z_std_inferred'].values)
 
-# +
-# Set a seed for reproducibility
-np.random.seed(852)
-
 # Get posterior samples of X_standardized
 w_samples_post =\
     w_dist_prior.rvs((num_dimensions, NUM_PRIOR_SAMPLES))
```

Comment:
This is an error. I should be sampling from the posterior distribution (i.e., `w_dist_post` and `z_dist_post`) not the prior distribution (i.e., `w_dist_prior` and `z_dist_prior`).

cc @bouzaghrane and @hassanobeid1994 for visibility.
Please note that these are the bugs that are of the highest priority to catch.

Reply:
I was actually going to bring this up in the mtg because I was confused. But glad you clarified it.

Reply:
Cool! btw, if you check the plots of the prior and posterior distributions of travel time, they now make much more sense. The prior plots are super diffuse, and the posterior plots are much more concentrated and better share the shape of the observed data.